src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,10 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
+        logger.info("Method login was called with login-> '{}'", login);
```

Comment:
Let's make our log messages informative. Include method parameters in the log message to make it more useful for debugging. For example: logger.info("Method login was called with login='{}'", login);

Reply:
method was called more suitable for DEBUG level